src/main/java/jnr/ffi/Struct.java

```diff
@@ -239,6 +245,34 @@ public double doubleValue() {
         }
     }
 
+    public static final class Alignment extends Number {
```

Comment:
I am a bit confused why this type is needed. Isn't this essentially just java.lang.Integer? Will we not want to support long alignments as well?